test-runner-frontend/src/Views/Dashboard.js

```diff
@@ -454,67 +201,70 @@ export default function Dashboard() {
+                sortDate={sortDate}
+              />
+            </Box>
           </Stack>
 
-          <Box sx={{ marginTop: '0.8%', marginBottom: '0.8%' }}>
-            <FormControl sx={{ minWidth: '10.2%' }} size="small">
-              <InputLabel>Test Runs</InputLabel>
-              <Select value={testRunSelection} onChange={handleChangeTestRunSelection} label="Test Runs">
-                <MenuItem value={'All Test Runs'}>All Test Runs</MenuItem>
-                <MenuItem data-testid="Master Only" value={'Master Only'}>
-                  Master Only
-                </MenuItem>
-              </Select>
-            </FormControl>
-
-            <FormControl sx={{ marginLeft: '1.2%', minWidth: '9%' }} size="small">
-              <InputLabel>Machine</InputLabel>
-              <Select value={machineSelection} onChange={handleChangeMachineSelection} label="Machine">
-                <MenuItem value={'All Machines'}>All Machines</MenuItem>
-                <MenuItem value={'e2-standard-32'}>e2-standard-32</MenuItem>
-                <MenuItem value={'e2-standard-2'}>e2-standard-2</MenuItem>
-              </Select>
-            </FormControl>
-          </Box>
-
-          {isCheckBoxActive === true ? (
-            checkBoxSelection.length > 2 ? (
-              <Alert data-testid="alert-warning" severity="warning" sx={{ width: '30%', marginTop: '1.6%' }}>
-                You can only select 2 test runs to compare!
-              </Alert>
-            ) : checkBoxSelection.length === 2 ? (
-              <Alert data-testid="alert-succes" severity="success" sx={{ width: '30%', marginTop: '1.6%' }}>
-                Nice selection! You can now compare this 2 test runs
-              </Alert>
-            ) : (
-              <Alert data-testid="alert-info" severity="info" sx={{ width: '30%', marginTop: '1.6%' }}>
-                Select {2 - checkBoxSelection.length} Test Runs to compare
-              </Alert>
-            )
-          ) : null}
-
-          <TestRunsTable
-            onCheckboxChange={onCheckboxChange}
-            isCheckBoxActive={isCheckBoxActive}
-            testRunResults={testRunResults}
-            sortDate={sortDate}
-          />
-        </Box>
-      </Stack>
-
-      <Box sx={{ width: '100%', bottom: '5%', position: 'absolute' }}>
-        <Typography variant="h7" sx={{ color: 'gray', left: '3%', position: 'absolute' }}>
-          <b>v1.0</b>
-        </Typography>
+          <Box sx={{ width: '100%', bottom: '5%', position: 'absolute' }}>
+            <Typography variant="h7" sx={{ color: 'gray', left: '3%', position: 'absolute' }}>
+              <b>v1.0</b>
+            </Typography>
 
-        <Typography variant="h7" sx={{ color: 'gray', left: '9%', position: 'absolute' }}>
-          Updated July, 2022
-        </Typography>
+            <Typography variant="h7" sx={{ color: 'gray', left: '9%', position: 'absolute' }}>
+              Updated August, 2022
```

Comment:
what is this label meant to represent? the last time the dashboard was built? the date from the last test run?

Reply:
I think it represents the last time the dashboard was built but, what do you think?

Reply:
hmm yeah im not too sure. i think we can probably remove this for the moment as:

1. if its representing the last time the dashboard was built, im not too sure if its that helpful
2. if it represents the last time the test runner was run, we can see based on results

Reply:
Yes, I agree. I'll remove it.